src/compute/leaf.rs

```diff
@@ -12,16 +13,43 @@ use crate::tree::LayoutTree;
 #[cfg(feature = "debug")]
 use crate::debug::NODE_LOGGER;
 
+/// The public interface to Taffy's Lead node algorithm implementation
+pub(crate) struct LeafAlgorithm;
+impl LayoutAlgorithm for LeafAlgorithm {
+    const NAME: &'static str = "LEAF";
+
+    fn perform_layout(
+        tree: &mut impl LayoutTree,
+        node: Node,
+        known_dimensions: Size<Option<f32>>,
+        parent_size: Size<Option<f32>>,
+        available_space: Size<AvailableSpace>,
+        sizing_mode: SizingMode,
+    ) -> SizeAndBaselines {
+        compute_generic(tree, node, known_dimensions, parent_size, available_space, sizing_mode)
+    }
+
+    fn measure_size(
+        tree: &mut impl LayoutTree,
+        node: Node,
+        known_dimensions: Size<Option<f32>>,
+        parent_size: Size<Option<f32>>,
+        available_space: Size<AvailableSpace>,
+        sizing_mode: SizingMode,
+    ) -> Size<f32> {
+        compute_generic(tree, node, known_dimensions, parent_size, available_space, sizing_mode).size
+    }
+}
+
 /// Compute the size of a leaf node (node with no children)
-pub(crate) fn compute(
+pub(crate) fn compute_generic(
```

Comment:
Ditto on name, and this is definitely making me suspicious that this should be in the trait.